### DISCUSSION TITLE
Cyberduck not resolving the host

### DISCUSSION THREAD

**Discussion by sdoigmm** [iterate-ch/cyberduck#14384] [Q&A] [answered]
**Describe the bug**

I downloaded Cyberduck today, it imported my sites from FileZilla. I tried to connect to one of those sites, in the status bar, it says:

---

Resolving example.sftp.wpengine.com

Connection failed

DNS lookup failed for example.sftp.wpengine.com

---

where the example is our correct sub subdomain.

FileZilla can resolve the host through DNS, but times out with version 3.63.2. I am downloading 3.63.2.1 at the moment, but the download of 14MB is taking 1 hour from the remote host.

**Expected behavior**
I expected CyberDuck to be able to resolve the remote host without a problem, given FileZilla can.

**Desktop**
 - macOS
 - Ventura 13.2.1

**Log Files**
The cyberduck.log is empty

**Comment by AliveDevil** [comment, marked as answer]
What does your latency/packet loss look like? 30 KBit/s (4 KByte/s) does look really slow, though.

When you did the test, and checked FileZilla after that, did you test Cyberduck again after FileZilla succeeding?

**Reply by sdoigmm** [reply]
Both FileZilla v3.63.2 and Cyberduck are connecting fine today.

Might of been an ISP issue.
